SplitTests/Integration/streaming/StreamingConnFail5xx.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  StreamingConnFail5xx.swift
+//  SplitTests
+//
+//  Created by Javier L. Avrudsky on 15/10/2020.
+//  Copyright © 2020 Split. All rights reserved.
+//
+
+import XCTest
+@testable import Split
+
+class StreamingConnFail5xxTest: XCTestCase {
```

Comment:
this file is named `StreamingConnFail5xx.swift`. shouldn't it be called `StreamingConnFail5xxTest.swift` like the class?

Reply:
This is some xcode magic. Gonna remove this files. Thanks doc!